IgOmeProfiling_pipeline.py

```diff
@@ -161,14 +166,18 @@ def run_pipeline(fastq_path, barcode2samplename_path, samplename2biologicalcondi
     parser.add_argument('-m', '--mapitope', action='store_true', help='use mapitope encoding')
 
     # optional parameters for the modelling step
+    parser.add_argument('--stop_random_forest', action='store_true', help='A boolean flag for mark if we need to run the random forest')
     parser.add_argument('--number_of_random_pssms', default=100, type=int, help='Number of pssm permutations')
+    parser.add_argument('--number_parallel_random_forest', default=20, type=int, help='How many random forest to run in parallel')
+    parser.add_argument('--min_value_error_random_forest', default=0, type=float, help='A min value for error that the run can stop in the random forest')
```

Comment:
Should be: A random forest model error value for convergence allowing to stop early